example_plugins/CMakeLists.txt

```diff
@@ -1,15 +1,19 @@
+# if compiling as an external component, point CMake to the containing folder
+
 CMAKE_MINIMUM_REQUIRED(VERSION 3.9 FATAL_ERROR)
 
 # Name the plugin project
-project(example_plugin LANGUAGES C CXX)
+project(example_plugins LANGUAGES C CXX)
 
 # Find the installed HOOMD
-find_package(HOOMD 2.6.0 REQUIRED)
+find_package(HOOMD 3.0.0 REQUIRED)
 
 message(STATUS "Found HOOMD ${HOOMD_VERSION}: ${HOOMD_INSTALL_PREFIX}/${PYTHON_SITE_INSTALL_DIR}")
 
 # Force installation to the HOOMD installation location
 set(CMAKE_INSTALL_PREFIX ${HOOMD_INSTALL_PREFIX} CACHE PATH "Installation prefix" FORCE)
 
 # add subdirectories
-add_subdirectory(${PROJECT_NAME})
+add_subdirectory(updater_plugin)
+add_subdirectory(pair_plugin)
+# add_subdirectory(force_compute_plugin)
```

Comment:
Yes, I wanted to suggest also implementing an example force compute since I have some experience doing this in my own projects. I have some uncommitted changes regarding that. But that can all be a separate pull request if you'd like to see it too. Maybe the example could be a replication of the example python ForceCompute from the docs, but in C++?

Reply:
Certainly, more examples are a welcome addition for the community. I don't have a preference for one pull request vs two on this. Either is fine, I leave it up to you.